webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -77,6 +79,7 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
     contextHolder: modalHyperparameterSearchContextHolder,
     modalOpen: openHyperparameterSearchModal,
   } = useModalHyperparameterSearch({ experiment });
+  const chartFlagOn = useFeature().isOn('chart');
```

Comment:
Should this be 'metrics_grid' per PR description?

Reply:
I updated as per discussion that there should be only one flag for all the recent chart related changes, see #5876.